include/tfm/tfm_builtin_key_ids.h

```diff
@@ -7,6 +7,25 @@
 #ifndef __TFM_BUILTIN_KEY_IDS_H__
 #define __TFM_BUILTIN_KEY_IDS_H__
 
+#ifdef CONFIG_PSA_CRYPTO_DRIVER_CRACEN
+
+#define CRACEN_BUILTIN_IDENTITY_KEY_ID ((uint32_t)0x7fffc001)
+#define CRACEN_BUILTIN_MKEK_ID	       ((uint32_t)0x7fffc002)
+#define CRACEN_BUILTIN_MEXT_ID	       ((uint32_t)0x7fffc003)
+
+#define CRACEN_IDENTITY_KEY_SLOT_NUMBER 0
+#define CRACEN_MKEK_SLOT_NUMBER		1
+#define CRACEN_MEXT_SLOT_NUMBER		2
+
+#define PSA_KEY_LOCATION_CRACEN (0x800000 | ('N' << 8))
```

Comment:
Is it intentional to not cast to psa_key_location_t like the non-CRACEN driver does in the else clause below?

Reply:
Not really, I'll add it when I do the fix mentioned in the above "issue"